src/main/resources/com/cloudbees/plugins/credentials/CredentialsSelectHelper/WrappedCredentialsStore/credential.jelly

```diff
@@ -24,21 +24,24 @@
  -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form" xmlns:l="/lib/layout">
-  <!-- TODO revert to dropdownDescriptorSelector when baseline is 1.645+ which supports `capture` attribute -->
-  <j:set var="descriptor" value="${it.credentialDescriptor}"/>
-  <j:set var="instance" value="${null}"/>
-  <f:dropdownList name="credentials" title="${%Kind}" help="${descriptor.getHelpFile('credentials')}">
-    <j:set var="current" value="${instance[credentials]}"/>
-    <j:set var="current" value="${current!=null ? current : null}"/>
-    <j:forEach var="descriptor" items="${descriptors}" varStatus="loop">
-      <f:dropdownListBlock value="${loop.index}" title="${descriptor.displayName}"
-        selected="${current.descriptor==descriptor || (current==null and descriptor==attrs.default)}" staplerClass="${descriptor.clazz.name}"
-        lazy="descriptor,it">
-        <l:ajax>
-          <j:set var="instance" value="${current.descriptor==descriptor ? current : null}" />
-          <st:include from="${descriptor}" page="${descriptor.configPage}" />
-        </l:ajax>
-      </f:dropdownListBlock>
-    </j:forEach>
-  </f:dropdownList>
+  <j:local>
+    <j:set var="it" value="${it.store}"/>
```

Comment:
Do you really need `j:local` here? AFAIK `it` would have been scoped only to this include anyway.

Reply:
I'm curious about this `j:local` -- it bleeds into the html that web browsers see. That isn't inherently bad, but in general one shouldn't bleed xml namespaced tags to html web browser.

Reply:
It shouldn’t, assuming the xml namespace is correctly defined

Reply:
I don't see a <local> tag in the Jelly documentation

Reply:
Ahh should be `scope` http://commons.apache.org/proper/commons-jelly/tags.html#core:scope Do you want to fix it?